shacl12-core/index.html

```diff
@@ -1539,8 +1539,13 @@ <h3>Property Shapes</h3>
 					<span data-syntax-rule="PropertyShape">A <dfn data-lt="property shapes">property shape</dfn> is a <a>shape</a> in the <a>shapes graph</a>
 					that is the <a>subject</a> of a <a>triple</a> that has <code>sh:path</code> as its <a>predicate</a>.</span>
 					<span data-syntax-rule="path-maxCount">A shape has at most one <a>value</a> for <code>sh:path</code>.</span>
-					<span data-syntax-rule="path-node">The <a>value</a> of <code>sh:path</code> in a property shape MUST be a <a>well-formed</a> <a>SHACL property path</a>
-					or (if it is not a <a>well-formed</a> <a>SHACL property path</a>) a <a>well-formed</a> <a>node expression</a>, but not a <a>literal</a>.</span>
+					<br/><br/>
+					<span data-syntax-rule="path-node">The <a>value</a> of <code>sh:path</code> in a property shape is either a <a>well-formed</a>
+					<a>SHACL property path</a> or a <a>well-formed</a> <a>node expression</a>.</span>
+					<span data-syntax-rule="path-expr-order">The syntax rules for <a>SHACL property paths</a> take precedence over <a>node expressions</a>,
+					for example so that <a>IRIs</a> are interpreted as <a>predicate path</a> instead of <a>IRI expression</a>.</span>
```

Comment:
```suggestion
					<span data-syntax-rule="path-expr-order">The syntax rules for <a>SHACL property paths</a> take precedence over <a>node expressions</a>;
					for example, this means that <a>IRIs</a> are interpreted as <a>predicate path</a> instead of <a>IRI expression</a>.</span>
```